skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -444,6 +447,73 @@ def _compute_metric_scores(
                 return score[0]
         return score
 
+    def _fit_time(self, **kwargs) -> Union[float, None]:
+        """Compute time to fit the estimator.
+
+        kwargs are available for compatibility with other metrics.
+        """
+        return self._parent._cache.get(self._parent._fit_time_cache_key())
+
+    def _predict_time(
+        self,
+        *,
+        data_source: DataSource = "test",
+        data_source_hash: Optional[int] = None,
+        X: Optional[ArrayLike] = None,
+        y: Optional[ArrayLike] = None,
+    ) -> Union[float, None]:
+        if data_source_hash is None:
+            X, _, data_source_hash = self._get_X_y_and_data_source_hash(
+                data_source=data_source, X=X, y=y
+            )
+
+        predict_time_cache_key = (
+            self._parent._hash,
+            data_source,
+            data_source_hash,
+            "predict_time",
+        )
+
+        return self._parent._cache.get(predict_time_cache_key)
```

Comment:
I know it is the default but it might be easier to have it such that we don't have any question if `get` is supposed to raise or just return `None`.

```suggestion
        return self._parent._cache.get(predict_time_cache_key, default=None)
```